integration-tests/http/__tests__/promotions/admin/promotions.spec.ts

```diff
@@ -495,24 +495,6 @@ medusaIntegrationTestRunner({
           )
         })
 
-        it("should throw an error when both campaign and campaign_id params are passed", async () => {
```

Comment:
@riqwan found this test and removed it as it doesn't apply anymore, but is it possible we need the `campaign`?

Reply:
We don't need it anymore. We built the APIs before the UI was ready and basically tried to handle all cases. This case turned out not to be used.